sim/midas/src/main/scala/midas/stage/GoldenGateCompilerPhase.scala

```diff
@@ -40,7 +40,7 @@ class GoldenGateCompilerPhase extends Phase {
 
     // Lower and emit simulator RTL and run user-requested host-transforms
     val hostLoweringCompiler = new Compiler(
-      Dependency[firrtl.VerilogEmitter] +:
+      Dependency[firrtl.SystemVerilogEmitter] +:
```

Comment:
This emitter is essentially a wrapper around the VerilogEmitter. However, since we append black boxes that do have SV constructs it was simplest to simply use this emitter and get the desired .sv suffix immediately. (You can't use the Verilog emitter and ask for a "<name>.sv" output file...)